lib/triplex.ex

```diff
@@ -177,12 +206,16 @@ defmodule Triplex do
   Returns all the tenants on the given `repo`.
   """
   def all(repo \\ config().repo) do
-    sql = """
-      SELECT schema_name
-      FROM information_schema.schemata
-      """
-    %Postgrex.Result{rows: result} = SQL.query!(repo, sql, [])
-
+    sql = case repo.__adapter__ do
+      Ecto.Adapters.MySQL ->
+        "SELECT name FROM #{config().tenant_table}"
+      _ ->
```

Comment:
Let's "let it fail" and put `Ecto.Adapters.Postgres` on this case clause ;)

Reply:
Yup